azure-pipelines-docs-integration.yml

```diff
@@ -119,5 +119,4 @@ stages:
             GCP_PROJECT: $(GCP_PROJECT)
             GCP_BIGQUERY_DATASET: $(GCP_BIGQUERY_DATASET)
             # Azure credentials
-            AZURE_ACCOUNT_URL: $(AZURE_ACCOUNT_URL)
-            AZURE_CREDENTIAL: $(AZURE_CREDENTIAL)
\ No newline at end of file
+            AZURE_CREDENTIAL: $(AZURE_CREDENTIAL)
```

Comment:
Originally put both the account url and credential as env vars but I later realized that the account url is not necessarily something that needs to be hidden. The account url needs to be in a particular format so showing it on our script is actual beneficial to the end user.

Reply:
Ah thanks for the explanation